blueoil/templates/lmnet/object_detection.tpl.py

```diff
@@ -107,7 +107,7 @@
 NETWORK.SCORE_THRESHOLD = score_threshold
 NETWORK.NMS_IOU_THRESHOLD = nms_iou_threshold
 NETWORK.NMS_MAX_OUTPUT_SIZE = nms_max_output_size
-NETWORK.SEEN_THRESHOLD = 8000
+NETWORK.LOSS_WARMUP_STEPS = 8000 / BATCH_SIZE
```

Comment:
Should we use `int(8000/BATCH_SIZE)`?

Reply:
yeah. I missed this one. I'll change that Thanks!